src/components/Auth/Input/InputPw.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import Input from './Input';
+import { AiFillEye } from 'react-icons/ai';
+import { AiFillEyeInvisible } from 'react-icons/ai';
+import { FaBan } from 'react-icons/fa';
+import { AiOutlineCheck } from 'react-icons/ai';
+import './InputPw.scss';
+
+const InputPw = props => {
+  const { id, name, inPutSet, value, innerInputText, ...others } = props;
+
+  //   // , inPutSet, ...others
+  const [isShowing, setIsShowing] = useState(true);
+
+  const onClick = () => {
+    setIsShowing(prev => !prev);
+  };
+
+  const islength = inPutSet.password.length > 7;
+  const rightPw = /^(?=.*[A-Za-z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,20})/;
+  const isPw = rightPw.test(inPutSet.password);
```

Comment:
여기도 변수명 조금 더 신경써주세요
```suggestion
  const isMinLength8 = inPutSet.password.length > 7;
  const passwordPattren = /^(?=.*[A-Za-z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,20})/;
  const isValidPassword = passwordPattren.test(inPutSet.password);
```
는 어떤가요?

Reply:
넵 바꿨습니다